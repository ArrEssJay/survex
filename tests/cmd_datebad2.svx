; invalid dates:
*begin
*date 1899.12.31
*date 1899.12.31
*date 1899.12
*date 1899
*date 1800
*date 1700.12.01
*date 100.01.01
*date 1899.02.03
*date 999.9.9
*date 1984.13.01
*date 1984.13.31
*date 1984.01.00
*date 1984.00.12
*date 1984.00
*date 1984.13
*date 1984.01.02-1984.01.01
*date 71-1970
*date 2079.01.01
*date 2079.12.31
*date 2099.06
*date 3033
*date 11974.02.03
*fix 1 0 0 0
1 2 1.0 020 -03
*end
