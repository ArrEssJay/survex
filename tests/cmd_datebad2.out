./cmd_datebad2.svx:4: warning: Invalid year (< 1900 or > 2078)
 *date 1899.12.31
./cmd_datebad2.svx:5: warning: Invalid year (< 1900 or > 2078)
 *date 1899.12.31
./cmd_datebad2.svx:6: warning: Invalid year (< 1900 or > 2078)
 *date 1899.12
./cmd_datebad2.svx:7: warning: Invalid year (< 1900 or > 2078)
 *date 1899
./cmd_datebad2.svx:8: warning: Invalid year (< 1900 or > 2078)
 *date 1800
./cmd_datebad2.svx:9: warning: Invalid year (< 1900 or > 2078)
 *date 1700.12.01
./cmd_datebad2.svx:10: warning: Invalid year (< 1900 or > 2078)
 *date 100.01.01
./cmd_datebad2.svx:11: warning: Invalid year (< 1900 or > 2078)
 *date 1899.02.03
./cmd_datebad2.svx:12: warning: Invalid year (< 1900 or > 2078)
 *date 999.9.9
./cmd_datebad2.svx:13: warning: Invalid month
 *date 1984.13.01
./cmd_datebad2.svx:14: warning: Invalid month
 *date 1984.13.31
./cmd_datebad2.svx:15: warning: Invalid day of the month
 *date 1984.01.00
./cmd_datebad2.svx:16: warning: Invalid month
 *date 1984.00.12
./cmd_datebad2.svx:17: warning: Invalid month
 *date 1984.00
./cmd_datebad2.svx:18: warning: Invalid month
 *date 1984.13
./cmd_datebad2.svx:19:28: error: End of date range is before the start
 *date 1984.01.02-1984.01.01
                            ^
./cmd_datebad2.svx:20:14: error: End of date range is before the start
 *date 71-1970
              ^
./cmd_datebad2.svx:21: warning: Invalid year (< 1900 or > 2078)
 *date 2079.01.01
./cmd_datebad2.svx:22: warning: Invalid year (< 1900 or > 2078)
 *date 2079.12.31
./cmd_datebad2.svx:23: warning: Invalid year (< 1900 or > 2078)
 *date 2099.06
./cmd_datebad2.svx:24: warning: Invalid year (< 1900 or > 2078)
 *date 3033
./cmd_datebad2.svx:25: warning: Invalid year (< 1900 or > 2078)
 *date 11974.02.03

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 2 survey stations, joined by 1 leg.
There are 0 loops.
Total length of survey legs =    1.00m (   1.00m adjusted)
Total plan length of survey legs =    1.00m
Total vertical length of survey legs =    0.05m
Vertical range = 0.05m (from 1 at 0.00m to 2 at -0.05m)
North-South range = 0.94m (from 2 at 0.94m to 1 at 0.00m)
East-West range = 0.34m (from 2 at 0.34m to 1 at 0.00m)
   2 1-nodes.
There were 20 warning(s) and 2 error(s) - no output files produced.
